cli/tests/test_messages_from_files.py

```diff
@@ -55,6 +55,26 @@ def test_messages_from_csv(omop_resources: Path) -> None:
     assert messages == expected_messages
 
 
+def test_messages_from_csv_multiple_projects(
+    omop_resources: Path, rows_in_session, mock_publisher
+) -> None:
+    """
+    GIVEN the database has a single Export entity, with one exported Image, one un-exported Image,
+    WHEN we parse a file with two Export entities, one new and one with two existing images and
+    one new image,
```

Comment:
Tried to make it a bit more clear, but not sure if it actually is

```suggestion
    WHEN we parse a file with two projects, each with the same 3 images
      where one project has already exported one of the images 
      and the other project has not exported any images
```

Reply:
nice, it's clearer to me (although I already know what the test does...)